src/fw-update/fw-update-device-interface.h

```diff
@@ -9,18 +9,15 @@
 
 namespace librealsense
 {
-    class updatable
+    class fw_checkable
```

Comment:
Pls rename to `firmware_image_check` or similar

Reply:
changed to "firmware_check_interface"